labkey-client-api/src/org/labkey/remoteapi/storage/DeleteCommand.java

```diff
@@ -0,0 +1,101 @@
+package org.labkey.remoteapi.storage;
+
+/**
+ * <p>
+ * Delete an existing LabKey Freezer Manager storage item. Note that deletion of freezers or locations within the
+ * freezer hierarchy will cascade the delete down the hierarchy to remove child locations and terminal storage locations.
```

Comment:
Do we want to make any note here about the behavior when samples are in storage?

Reply:
done